conf/app.routes

```diff
@@ -10,7 +10,7 @@ GET        /apply-for-help-to-save/how-the-account-works    @uk.gov.hmrc.helptos
 GET        /apply-for-help-to-save/how-we-calculate-bonuses @uk.gov.hmrc.helptosavefrontend.controllers.IntroductionController.getHowWeCalculateBonuses
 GET        /apply-for-help-to-save/apply                    @uk.gov.hmrc.helptosavefrontend.controllers.IntroductionController.getApply
 
-GET        /check-and-confirm-your-email                    @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.getConfirmEmailPage
+GET        /select-email                                    @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.getConfirmEmailPage
```

Comment:
if you change this, you should change all its occurances in the message file as well, for ex:
`hts.check-and-confirm-your-email.title`